pvanalytics/util/_fit.py

```diff
@@ -58,8 +59,10 @@ def quartic_restricted(data, noon=720):
 
     Parameters
     ----------
-    data : Series
-        power or irradiance data.
+    x : Series
+        x values for data in `y`
+    y : Series
```

Comment:
```suggestion
    y : array or Series
```

Reply:
As implemented, `y` actually needs to be a Series (we use `y.median()` for one of the restrictions).